ios/FluentUI/Notification/NotificationTokenSet.swift

```diff
@@ -166,9 +166,9 @@ public class NotificationTokenSet: ControlTokenSet<NotificationTokenSet.Tokens>
                 return .float {
                     switch style().isToast {
                     case true:
-                        return GlobalTokens.borderRadius(.xLarge)
+                        return GlobalTokens.borderRadius(.cornerRadius120)
                     case false:
-                        return GlobalTokens.borderSize(.none)
+                        return GlobalTokens.borderSize(.strokeWidthNone)
```

Comment:
Could you change the false case to pull the zero value from the same set of tokens as the non-zero value?

Reply:
To clarify: they should both be borderRadius if possible